aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -569,7 +687,7 @@ std::string OperatorEntry::dumpComputedTable() const {
   // Need to handle Undefined separately, because its a runtime key that can't be represented
   // in a DispatchKeySet.
   std::vector<DispatchKey> runtime_keys = {DispatchKey::Undefined};
```

Comment:
Can reserve the std::vector size?

Reply:
this looks like a debugging function, probably not worth the attention beyond the blanket find/replace I did. if I were to pay attention to it, I would just extract the loop body into a lambda and then call it first for undefined before looping over allDispatchKeysInFullset